keras_cv/backend/random.py

```diff
@@ -36,6 +38,13 @@ def next(self, ordered=True):
             self._current_seed[0] += 1
             return self._current_seed[:]
 
+    def get_config(self):
+        return {"seed": self._seed, **self._kwargs}
```

Comment:
you would not need `**self._kwargs`

Reply:
Why?  If the user passes kwargs we want to capture that in the config that's serializied.

Reply:
adding reference from keras 3 here - https://github.com/keras-team/keras/pull/18933

Reply:
The reconstructed object will not be same as one originally created if we don't serialize the kwargs as well

Reply:
Resolving based on offline discussion. Adding `kwargs` to get_config is not recommended because we have no way to guarantee the provided args would be serializable or not.